site-cookbooks/jenkins-slave/recipes/default.rb

```diff
@@ -97,7 +97,7 @@
 docker_installation_package 'default' do
   action :create
   package_name 'docker.io'
-  package_version '1.13.1-0ubuntu1~16.04.2'
+  package_version '17.12.1-0ubuntu1'
```

Comment:
this needs if-elseing with a versionconstraint so xenial arm continues to be cookable.